public/stylesheets/openqa.css

```diff
@@ -533,6 +533,8 @@ table.overview .failedmodule {
 .fa.result_incomplete { color: #AF1E11;}
 .fa.module_none { color:#bebebe;}
 
+.comment { color: Grey; }
```

Comment:
I think you want the selector more specific - or the class name itself. comments can be anywhere anything - but you're only meaning the one icon on the overview page.